src/ConfigCannotBeCachedException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-config-aggregator for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-config-aggregator/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-config-aggregator/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ConfigAggregator;
+
+use Brick\VarExporter\ExportException;
+use RuntimeException;
+
+use function sprintf;
+
+class ConfigCannotBeCachedException extends RuntimeException
+{
+    /**
+     * @param ExportException $exportException
+     *
+     * @return ConfigCannotBeCachedException
+     */
+    public static function fromExporterException(ExportException $exportException)
+    {
+        return new self(
+            sprintf(
+                'Cannot export config into a cache file. Config contains uncacheable entries: %s',
```

Comment:
Should the underlying exception message leak into the message?

Reply:
Does it contain information about the uncacheable entries? And is there any other way to get at that information?

If the answer to the first question is yes and the second is no, then this is the right approach, from a "can I diagnose what is wrong from the exception message?" perspective.

Reply:
The underlying exception message contains useful debug information, so I'll leave it in the message.